blocks/hero/hero.css

```diff
@@ -96,7 +96,7 @@
 
         & .title {
             text-shadow: 1px 3px 4px rgba(0 0 0 / 30%);
-            font-size: 2em;
+            font-size: 1.6em;
```

Comment:
This size still isn't right. You'll probably need 2 or 3 different definitions , one per break point. Start with the smallest font size as the default, then use `min-width: <breakpoint>` media queries to make it larger on larger screens.